---
title: MSSQLSERVER_8680 | Microsoft Docs
ms.custom: ''
ms.date: 03/06/2017
ms.prod: sql-server-2014
ms.reviewer: ''
ms.suite: ''
ms.technology:
- database-engine
ms.tgt_pltfrm: ''
ms.topic: article
helpviewer_keywords:
- 8680 (Database Engine error)
ms.assetid: 79f7270e-e4df-4f4e-bc78-2dcad6a9f140
caps.latest.revision: 13
author: craigg-msft
ms.author: craigg
manager: jhubbard
ms.openlocfilehash: f2cbb011ace5c2af59e9e27fa415a1bd42ac5cdf
ms.sourcegitcommit: 5dd5cad0c1bbd308471d6c885f516948ad67dfcf
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 06/19/2018
ms.locfileid: "36076600"
---
# <a name="mssqlserver8680"></a>MSSQLSERVER_8680
    
## <a name="details"></a>詳細  
  
|||  
|-|-|  
|製品名|SQL Server|  
|イベント ID|8680|  
|イベント ソース|MSSQLSERVER|  
|コンポーネント|SQLEngine|  
|シンボル名|REMOTE_ERR|  
|メッセージ テキスト|内部クエリ プロセッサ エラー: リモート クエリ フェーズを処理中に、クエリ プロセッサで予期しないエラーが発生しました。|  
  
## <a name="explanation"></a>説明  
 クエリ処理中に内部エラーが発生しました。  
  
## <a name="user-action"></a>ユーザーの操作  
 マイクロソフト カスタマー サポート サービス (CSS) に連絡し、エラーの説明とトレースまたはダンプ (取得可能な場合) を報告してください。  
  
  